server/interceptor/interceptor.go

```diff
@@ -102,6 +106,10 @@ func initMetrics(mf monitoring.MetricFactory) {
 		"interceptor_request_denied_count",
 		"Number of requests by denied, labeled according to the reason for denial",
 		"reason", monitoring.TreeIDLabel, "quota_user")
+	contextErrCounter = mf.NewCounter(
+		"context_err_counter",
```

Comment:
"interceptor_context_err_count", so it has a clearer scope and the same suffix as other metrics?

Do you think this is an interesting metric to track? Won't is just show up as cancelled RPCs anyway?

Reply:
Renamed.

At least for the moment I'd like to be able to see which stage is slow and try to correlate it with the existing rpc client metrics.